sql/hive/src/test/java/org/apache/spark/sql/hive/test/Complex.java

```diff
@@ -16,7 +16,7 @@
  */
 package org.apache.spark.sql.hive.test;
 
-import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
```

Comment:
I think this can be submitted as an independent pr. Also, do you know why check-style didn't detect this issue?

https://github.com/apache/spark/blob/9f34b8eca2f3578367352ec28532e29fe3a91864/dev/checkstyle.xml#L199-L202

Reply:
not sure, why checkstyle does not complain about that. let me fix it independently.

Reply:
opened https://github.com/apache/spark/pull/46154